src/Qowaiv.Validation.Abstractions/Result.cs

```diff
@@ -18,8 +18,21 @@ internal Result(FixedMessages messages)
     [Pure]
     public bool IsValid => !Errors.Any();
 
-    /// <summary>Gets all messages with <see cref="ValidationSeverity.Error"/>.</summary>
+    /// <summary>Returns a result where all warnings are converted into error messages.</summary>
+    /// <remarks>
+    /// The type of the messages that where warnings might change due to this.
+    /// </remarks>
+    [Pure]
+    public Result WarningsAsErrors() 
+        => new(FixedMessages.Empty.AddRange(Messages.Select(WarningAsError)));
 
+    [Pure]
+    static internal IValidationMessage WarningAsError(IValidationMessage message)
```

Comment:
Is it possible to exclude certain messages from the transformation to Error? F.E. our "Alerts" are also tagged as severity Warning but should never be an Error

Reply:
Do you think that should be provided by this library? You can write your custom converter anyway?!